kuma/wiki/inline_examples.py

```diff
@@ -0,0 +1,237 @@
+INLINE_EXAMPLES = {
+    "FOR_EACH": """
+<div class="interactive interactive-js">
+<link href="/static/styles/libs/interactive-examples/codemirror-5-31-0.css" rel="stylesheet" />
+<link href="/static/styles/libs/interactive-examples/editor-js.css" rel="stylesheet" />
+
+<script>"use strict";function postToKuma(e){postMessage(e,"https://developer.mozilla.org")}postToKuma({markName:"interactive-editor-loading"}),document.addEventListener("readystatechange",function(e){switch(e.target.readyState){case"interactive":postToKuma({markName:"interactive-editor-interactive",measureName:"ie-time-to-interactive",startMark:"interactive-editor-loading",endMark:"interactive-editor-interactive"});break;case"complete":postToKuma({markName:"interactive-editor-complete",measureName:"ie-time-to-complete",startMark:"interactive-editor-loading",endMark:"interactive-editor-complete"})}});</script>
```

Comment:
I get this error locally:

> Failed to execute ‘postMessage’ on ‘DOMWindow’: The target origin provided (‘https://developer.mozilla.org’) does not match the recipient window’s origin (‘http://localhost:8000’).

This could be changed by using ``settings.SITE_URL``. It is probably OK for this experiment.